plugin-animation/src/main/java/com/mapbox/maps/plugin/animation/animator/CameraAnimator.kt

```diff
@@ -93,6 +98,54 @@ abstract class CameraAnimator<out T> (
     }
   }
 
+  /**
+   * The most recent value calculated by this <code>ValueAnimator</code> when there is just one
+   * property being animated. This value is only sensible while the animation is running. The main
+   * purpose for this read-only property is to retrieve the value from the <code>ValueAnimator</code>
+   * during a call to {@link AnimatorUpdateListener#onAnimationUpdate(ValueAnimator)}, which
+   * is called during each animation frame, immediately after the value is calculated.
+   *
+   * @return animatedValue The value most recently calculated by this <code>ValueAnimator</code> for
+   * the single property being animated. If there are several properties being animated
+   * (specified by several PropertyValuesHolder objects in the constructor), this function
+   * returns the animated value for the first of those objects.
+   */
+  override fun getAnimatedValue(): Any {
+    // For immediate animations on API 23, as we introduced the bypass logic in handleImmediateAnimationOnAPI23(),
+    // the returned ValueAnimator.getAnimatedValue() will be null, in this case, we should return the
+    // last configured target value, so we immediately jump to the target value.
+    return super.getAnimatedValue() ?: targets.last() as Any
+  }
+
+  /**
+   * Handle immediate animation(when duration and startDelay of the animation is 0) on devices running
+   * API 23 or below, by sending updates to animator listeners directly without triggering [ValueAnimator]'s start().
+   *
+   * @return true if the animation is handled, false otherwise
+   */
+  private fun handleImmediateAnimationOnAPI23(): Boolean {
```

Comment:
```suggestion
  private fun handleImmediateAnimationOnAPI23OrBelow(): Boolean {
```

Reply:
technically our minimum supported API level is 23 🙈

Reply:
right, I thought our minimum supported API is 23, but it's 21 indeed, will fix

Reply:
https://github.com/mapbox/mapbox-maps-android/blob/main/buildSrc/src/main/kotlin/Project.kt#L2
okay :wink: